### ISSUE TITLE
Create card lists

### ISSUE THREAD

**Issue by ajlmacmillan** [ajlmacmillan/ajlmacmillan.github.com#5]
Cards should be structured as unordered lists to help with accessibility and screen readers.

Update the .cards-container to be <ul> and the JS block that produces the card to wrap .card in <li>
